src/unit/test_sds.c

```diff
@@ -252,50 +252,55 @@ int test_sds(int argc, char **argv, int flags) {
     return 0;
 }
 
+size_t sdsRealAllocSize(sds s) {
+    return sdsAllocSize(s) + PREFIX_SIZE;
+}
+
```

Comment:
From a modularization perspective, we could add this as a function in zmalloc.c and forward declare it here to access it for tests but not expose it in the header file. I don't know if I prefer that approach though.